tests/comparison.php

```diff
@@ -14,3 +14,12 @@
     ok($rgb->equals($hex), 'hex can be compared to RGB');
     ok(! $hex->equals($differentColor), 'different colors do not match');
 });
+
+test('it can compare two colors', function () {
+    $first = Color::hex('#aaa');
+    $second = Color::hex('#aaa');
+    $differentColor = Color::hex('#bbb');
+
+    ok(Color::bothEqual($first, $second), 'two colors can be compared');
+    ok(! Color::bothEqual($first, $differentColor), 'different colors do not match');
```

Comment:
If `Color::bothEqual` returns a `bool`, I prefer doing a strict comparison instead of a generic truthy / falsy test condition.
```suggestion
    ok(Color::bothEqual($first, $second) === true, 'two colors can be compared');
    ok(Color::bothEqual($first, $differentColor) === false, 'different colors do not match');
```